From 1245b63d8af84928936f832f5396ff38d6129dab Mon Sep 17 00:00:00 2001
From: Yuwei Chen <yuwei.chen@intel.com>
Date: Fri, 21 Aug 2020 15:46:12 +0800
Subject: [PATCH] Test case for Library - CpuSleep.asm file

Description: Change the function in the MdePkg/Library/BaseCpuLib/Arm/CpuSleep.asm.

Repo: edk2

Signed-off-by: Yuwei Chen <yuwei.chen@intel.com>
---
 MdePkg/Library/BaseCpuLib/Arm/CpuSleep.asm | 3 ++-
 1 file changed, 2 insertions(+), 1 deletion(-)

diff --git a/MdePkg/Library/BaseCpuLib/Arm/CpuSleep.asm b/MdePkg/Library/BaseCpuLib/Arm/CpuSleep.asm
index ae32705a69..132d6cc57d 100644
--- a/MdePkg/Library/BaseCpuLib/Arm/CpuSleep.asm
+++ b/MdePkg/Library/BaseCpuLib/Arm/CpuSleep.asm
@@ -32,7 +32,8 @@
 ;  );
 ;
 CpuSleep
-    WFI
+    MOV r0,#2
+	WFI
     BX LR
 
   END
-- 
2.27.0.windows.1

